### ISSUE TITLE
Compatability with Alias Isolation?

### ISSUE THREAD

**Issue by SimplexPL** [Nibre/MotherVR#13]
https://github.com/aliasIsolation/aliasIsolation

This mod greatly improves graphics quality by removing shimmering/pixel crawling/aliasing which is even more pronounced in VR. It would be great if those mods could be run at the same time.

https://www.youtube.com/watch?v=G6Aq7Ayoqvo

**Comment by bakman2** [comment]
> No support for the unofficial Rift mode.
This was the rift-mode that was build-in by Sega.

Did you try it ?

**Comment by SimplexPL** [comment]
You are right. I read "mod" instead of mode. 
I did not try it because I don't have a Rift.

**Comment by traveltrousers** [comment]
> This was the rift-mode that was build-in by Sega.

I don't think so.... the code is here : [https://github.com/aliasIsolation/aliasIsolation](url) so it might be reasonably easy to merge into MotherVR

**Comment by bakman2** [comment]
>I don't think so
I think so because that mod is over a year old.  

it _might_ work, did you try it ? I don't have visual studio.

**Comment by traveltrousers** [comment]
Yeah, I tried it.... the removal of jaggies is huge.... the game as is looks pretty nasty.

You don't need VS, just get the release : https://github.com/aliasIsolation/aliasIsolation/releases

**Comment by Nibre** [comment]
This is something that's in the works

**Comment by SputnikKaputtnik** [comment]
is it worth trying with 0.6.0 now?

**Comment by olgrea** [comment]
I'm wondering that too. I'm waiting for Alias Isolation integration before playing the game in VR just so I can have the best experience possible

**Comment by JoshWobbles** [comment]
Has anyone given it a shot lately?

**Comment by Midnightmaster** [comment]
> 
> 
> Bummer, the aliasing is the only thing keeping this from being the most graphically impressive vr game there is imho.

Try -pixelscale 4.0 with a Nvidia 2080ti, i think the aliasing is gone then... ;-)

**Comment by thehhmdb** [comment]
The alias solution would make it look much better, the jaggies are really bad

**Comment by spyro2000** [comment]
So, this couldn't be merged then? Was hoping this two mods could be combined.
